### PR TITLE
fix(gzip): schema typo

### LINKED ISSUE #5085: bug: gzip plugin schema typo error
### Issue description

Mistakenly write minitems as minitem in the schema of the gzip plugin.
[go to that line](https://github.com/apache/apisix/blob/c907ef0f663eba5f10686e13e431bd007e7c5ac3/apisix/plugins/gzip.lua#L34)

### Environment

- apisix version (cmd: `apisix version`): 2.9
- OS (cmd: `uname -a`): Linux d4af861caf32 5.10.16.3-microsoft-standard-WSL2 #1 SMP Fri Apr 2 22:23:49 UTC 2021 x86_64 Linux
- OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`): openresty/1.19.3.1
- etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API): 3.4.15
- apisix-dashboard version, if have: 2.8
- the plugin runner version, if the issue is about a plugin runner (cmd: depended on the kind of runner): none
- luarocks version, if the issue is about installation (cmd: `luarocks --version`): 3.7.0

### Steps to reproduce

1. An error occurs when JSON schema compiles the schema document output by APISIX Control API.
eg. unknown keyword `minItem`

### Actual result

return `minItem`

### Error log

Unhandled Rejection (Error): strict mode: unknown keyword: "minItem" by JavaScript

### Expected result

return `minItems`

### PR DESCRIPTION
**Pull request by bzp2010**
### What this PR does / why we need it:
fix #5085 

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

**Review by spacewander — COMMENTED** [review]
Could you add a test in https://github.com/apache/apisix/blob/master/t/plugin/gzip.t like https://github.com/apache/apisix/blob/1a072f4271e74e27339823fec5817e1c2d42c719/t/plugin/real-ip.t#L49?

**Comment by bzp2010** [comment]
> Could you add a test in https://github.com/apache/apisix/blob/master/t/plugin/gzip.t like
> 
> https://github.com/apache/apisix/blob/1a072f4271e74e27339823fec5817e1c2d42c719/t/plugin/real-ip.t#L49
> 
> ?

Ok, I will add it now.

File: t/plugin/gzip.t [hunk 1]
```diff
@@ -46,7 +46,57 @@ run_tests();
 
 __DATA__
 
-=== TEST 1: sanity
```
Comment (spacewander) [inline t/plugin/gzip.t:49]: Please add new test from the bottom

Comment (bzp2010) [reply t/plugin/gzip.t]: fixed

**Review by tzssangglass — APPROVED** [review]

**Review by nic-chen — APPROVED** [review]

**Comment by bzp2010** [comment]
Hi, 
I noticed that other CI build tests passed, and only `CI / build (ubuntu-18.04, linux_openresty)` failed. Does this have anything to do with my change?

**Review by shuaijinchao — APPROVED** [review]

**Review by tokers — APPROVED** [review]

**Comment by spacewander** [comment]
> 
> 
> Hi,
> I noticed that other CI build tests passed, and only `CI / build (ubuntu-18.04, linux_openresty)` failed. Does this have anything to do with my change?

Yes, please check the output of the CI log:
```
#   Failed test 'TEST 19: schema check - response_body - response is expected (repeated req 0, req 0)'
#   at /home/runner/work/apisix/apisix/test-nginx/lib/Test/Nginx/Socket.pm line 1589.
# @@ -1,6 +1,6 @@
#  {"error_msg":"failed to check the configuration of plugin gzip err: property \"types\" validation failed: object matches none of the requireds"}
#  {"error_msg":"failed to check the configuration of plugin gzip err: property \"min_length\" validation failed: expected 0 to be greater than 1"}
# -{"error_msg":"failed to check the configuration of plugin gzip err: property \"comp_level\" validation failed: expected 0 to be greater than 1"}
# +{"error_msg":"failed to check the configuration of plugin gzip err: property \"comp_level\" validation failed: expected 10 to be smaller than 9"}
#  {"error_msg":"failed to check the configuration of plugin gzip err: property \"http_version\" validation failed: matches none of the enum values"}
#  {"error_msg":"failed to check the configuration of plugin gzip err: property \"buffers\" validation failed: property \"number\" validation failed: expected 0 to be greater than 1"}
#  {"error_msg":"failed to check the configuration of plugin gzip err: property \"vary\" validation failed: wrong type: expected boolean, got number"}
```

**Comment by bzp2010** [comment]
Thank you very much for pointing out that problem. It was modified during the adjustment sequence, but it was not completely modified correctly.

**Pull request merged**